pkg/controller/core/workload_controller.go

```diff
@@ -462,7 +460,7 @@ func (r *WorkloadReconciler) reconcileCheckBasedEviction(ctx context.Context, wl
 	workload.ResetChecksOnEviction(wl, r.clock.Now())
 	reportWorkloadEvictedOnce := workload.WorkloadEvictionStateInc(wl, kueue.WorkloadEvictedByAdmissionCheck, "")
 	if err := workload.ApplyAdmissionStatus(ctx, r.client, wl, true, r.clock); err != nil {
-		return false, err
+		return false, client.IgnoreNotFound(err)
```

Comment:
The same issue exists here, as well as in other functions.

Reply:
https://github.com/kubernetes-sigs/kueue/pull/5832#discussion_r2179211962

Reply:
@mbobrovskyi Fixed. PTAL.😄